The avaraging tests seems to be flaky due to rounding errors. Test fails on
x86 and s390x

https://github.com/ImageMagick/ImageMagick/issues/1576#issuecomment-494595404

diff --git a/Magick++/tests/tests.tap b/Magick++/tests/tests.tap
index b5c15ff..bb83980 100755
--- a/Magick++/tests/tests.tap
+++ b/Magick++/tests/tests.tap
@@ -8,14 +8,14 @@
 #
 subdir=Magick++/tests
 . ./common.shi
-echo "1..13"
+echo "1..12"
 
 SRCDIR=${top_srcdir}/${subdir}/
 export SRCDIR
 
 cd ${subdir} || exit 1
 
-for mytest in appendImages attributes averageImages coalesceImages coderInfo color colorHistogram exceptions geometry montageImages morphImages readWriteBlob readWriteImages
+for mytest in appendImages attributes coalesceImages coderInfo color colorHistogram exceptions geometry montageImages morphImages readWriteBlob readWriteImages
 do
   ./${mytest} && echo "ok" || echo "not ok"
 done
